.github/workflows/promote-bundle.yaml

```diff
@@ -0,0 +1,33 @@
+name: promote-bundle
+
+on:
+  workflow_dispatch:
+    inputs:
+      bundle_variant:
+        required: true
+        type: choice
+        options:
+          - SDCORE
+          - SDCORE_CONTROL_PLANE
+          - SDCORE_USER_PLANE
+        description: "SD-Core bundle type"
+      channel:
+        required: false
+        type: choice
+        options:
+          - beta
+          - candidate
+          - stable
+        default: edge
+        description: "Channel to promote to"
```

Comment:
I don't think it makes sense to promote to edge by default. There is no such thing as promoting to edge.

Reply:
Yes, the default value wasn't supposed to be here at all. Copy-paste error.
Fixed.